google/cloud/datastore_admin_v1/services/datastore_admin/client.py

```diff
@@ -646,40 +658,175 @@ def import_entities(
         response = operation.from_gapic(
             response,
             self._transport.operations_client,
-            empty.Empty,
+            empty_pb2.Empty,
             metadata_type=datastore_admin.ImportEntitiesMetadata,
         )
 
         # Done; return the response.
         return response
 
+    def create_index(
+        self,
+        request: Union[datastore_admin.CreateIndexRequest, dict] = None,
+        *,
+        retry: OptionalRetry = gapic_v1.method.DEFAULT,
+        timeout: float = None,
+        metadata: Sequence[Tuple[str, str]] = (),
+    ) -> operation.Operation:
+        r"""Creates the specified index. A newly created index's initial
```

Comment:
This looks like a brand new API method - should it be mentioned in the release notes?

Reply:
Great catch! Done.